content/providers/provider-aws/authentication.md

```diff
@@ -0,0 +1,579 @@
+    region: us-east-2
+  providerConfigRef:
+    name: key-based-providerconfig
+```
+
+### Role chaining
+
+To use 
+[AWS IAM role chaining](https://docs.aws.amazon.com/IAM/latest/UserGuide/id_roles_terms-and-concepts.html#iam-term-role-chaining)
+add a 
+{{<hover label="keychains" line="12">}}assumeRoleChain{{</hover>}} object to the 
+{{<hover label="keychains" line="2">}}ProviderConfig{{</hover>}}.
+
+Inside the {{<hover label="keychains" line="12">}}assumeRoleChain{{</hover>}}
+list one or more roles to assume, in order. 
+
+```yaml {label="keychains"}
+apiVersion: aws.upbound.io/v1beta1
+kind: ProviderConfig
+metadata:
+  name: default
+spec:
+  credentials:
+    source: Secret
+    secretRef:
+      namespace: crossplane-system
+      name: aws-secret
+      key: my-aws-secret
+  assumeRoleChain:
+    - roleARN: "arn:aws:iam::111122223333:role/my-custom-role"
+```
+
+<!-- vale off -->
+## WebIdentity
+<!-- vale on -->
+When running the AWS Provider in an Amazon managed Kubernetes cluster (`EKS`)
+the Provider may use 
+[AssumeRoleWithWebIdentity](https://docs.aws.amazon.com/STS/latest/APIReference/API_AssumeRoleWithWebIdentity.html)
+for authentication. 
+
+WebIdentity uses an OpenID Connect ID token to authenticate and use a specific
+AWS IAM role.
+
+{{<hint "important">}}
+WebIdentity is only supported with Crossplane running in Amazon managed
+Kubernetes clusters (`EKS`).
+{{< /hint >}}
+
+Configuring WebIdentity with the AWS Provider requires:
+* an AWS 
+[IAM OIDC Provider](https://docs.aws.amazon.com/eks/latest/userguide/enable-iam-roles-for-service-accounts.html)
+* an AWS IAM Role with an editable [trust policy](https://docs.aws.amazon.com/IAM/latest/UserGuide/id_roles_terms-and-concepts.html#term_trust-policy)
+* a ProviderConfig to enable WebIdentity authentication
+
+### Create an IAM OIDC provider
+
+WebIdentity relies on the EKS cluster OIDC provider. 
+
+Follow the [AWS instructions](https://docs.aws.amazon.com/eks/latest/userguide/enable-iam-roles-for-service-accounts.html) 
+to create an _IAM OIDC provider_ with your _EKS OIDC provider URL_. 
+
+### Edit the IAM role
+
+Supporting WebIdentity requires matching the EKS OIDC information to the 
+specific role through a role trust policy. 
+
+{{<hint "tip" >}}
+Read the [AWS trust policies blog](https://aws.amazon.com/blogs/security/how-to-use-trust-policies-with-iam-roles/) 
+for more information on trust policies.
+{{< /hint >}}
+
+The trust policy references the OIDC provider ARN and the provider AWS pod name.
+
+In the policy {{<hover label="trust" line="6">}}Principal{{</hover>}} enter
+{{<hover label="trust" line="7">}}"Federated": "&lt;OIDC_PROVIDER_ARN&gt;"{{</hover>}}.
+
+Add a {{<hover label="trust" line="10">}}Condition{{</hover>}} to restrict
+access to the role to only the Provider pod.  
+
+The {{<hover label="trust" line="10">}}Condition{{</hover>}} uses 
+{{<hover label="trust" line="11">}}StringLike{{</hover>}} to generically match
+the Provider pod name.
```

Comment:
The IRSA IAM Role IdentityTrust relies on the ServiceAccount Name rather than the Pod Name. 
 
```
system:serviceaccount:$namespace:$service_account```
```

Reply:
fixed, thank you!